test/ui/presets.test.js

```diff
@@ -0,0 +1,205 @@
+const timeout = process.env.SLOWMO ? 30000 : 100000;
+beforeAll(async () => {
+  await page.goto('http://127.0.0.1:8080/index.html', {waitUntil: 'domcontentloaded'});
```

Comment:
Are we sure that the port number should be 8080?

Reply:
Maybe that's just because you're running a local server at 8080? For example, here, we explicitly set up a test server running at port 3000:

https://github.com/publiclab/image-sequencer/blob/1618a97b82480485299cba8c36f85449251494b8/jest-puppeteer.config.js#L6-L10

Reply:
You know, i think we can just do this locally, can't we? No server? What about:

```suggestion
  await page.goto('index.html', {waitUntil: 'domcontentloaded'});
```

? Let me try...